src/data.js

```diff
@@ -28,6 +28,16 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Matt Holovach',
+    description: 'Loves SEO, improving coding skills and good food',
+    url: 'https://www.coloradoseodesign.com/uses.php',
+    emoji: '✝',
+    country: 'us',
+    computer: 'microsoft',
```

Comment:
@jorgeruvalcaba you mean

```suggestion
    computer: 'windows',
```

? it is `microsoft`, but that is wrong :D